.github/ISSUE_TEMPLATE/DesignDocIssue

```diff
@@ -0,0 +1,12 @@
+---
+name: Design Doc Project Issue Template
+about: A template to follow when creating a new issue related to a design doc project
+---
+
+# Part of the <Project Name> project
+Main issue: <Issue Link>
+Doc: <Doc Link>
+Project: <Project Link>
+
+# <Feature Description>
+<!-- Describe the section of the doc that this issue is covering -->
```

Comment:
Maybe we could add manual and automated testing steps for the feature in the issue too? Maybe make them optional because that could be useful to give proper scope to an engineer grabbing the issue w/o as much context.